### PR TITLE
Entity >Board: Create a new "Pending" record

### PR DESCRIPTION
**Pull request by MariaMoroz**
https://trello.com/c/jLPR2Zbn/457-at00101javascript-entity-board-create-a-new-record

### CODE REVIEW

File: test/specs/us001_01.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+describe('Login Page', () => {
+    before(() => {
+        browser.maximizeWindow();
+        browser.url('/');
+    })
+
+    it('01/LP-Should verify login name IF accept valid data', () => {
+        const loginName = 'tester13@tester.test';
+        const element = "//input[@class='form-control']";
+        expect($(element).setValue(loginName));
```
Comment (dashasandler) [inline test/specs/us001_01.js:12]: If you do not use 'to.eq' you do not need to use 'expect'

File: test/specs/us001_01.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    });
+    it('02/LP-Should verify password IF accept valid data', () => {
+        const password = '7VEyv9EKaZ';
+        const passElement = "//input[@class='form-control']";
+        expect($$(passElement)[1].setValue(password));
```
Comment (dashasandler) [inline test/specs/us001_01.js:17]: If you do not use 'to.eq' you do not need to use 'expect'

File: test/specs/us001_01.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    });
+    it('03/LP-Should click sign in btn', () => {
+        const singInBtn = "//button[@class='btn btn-lg btn-primary btn-block']";
+        $(singInBtn).click();
+        const homeBtn = '//a[@class="navbar-brand"]'
```
Comment (dashasandler) [inline test/specs/us001_01.js:22]: missing ;

File: test/specs/us001_01.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+        expect($(homeBtn).getText()).to.equal('Home');
+    });
+    it('04/LP-Should reset all records', () => {
+        const menuReset = $('#navbarDropdownProfile');
+        menuReset.click();
+        const reset = "//a[text()='!!! Reset all for my user !!!']"
```
Comment (dashasandler) [inline test/specs/us001_01.js:28]: missing ;

File: test/specs/us001_01.js (hunk 1 above)
Comment (MariaMoroz) [reply test/specs/us001_01.js]: thank you, done

File: test/specs/us001_01.js (hunk 2 above)
Comment (MariaMoroz) [reply test/specs/us001_01.js]: done

File: test/specs/us001_01.js (hunk 3 above)
Comment (MariaMoroz) [reply test/specs/us001_01.js]: done

File: test/specs/us001_01.js (hunk 4 above)
Comment (MariaMoroz) [reply test/specs/us001_01.js]: done

File: test/specs/us001_01.js [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+        for(var element=0; element < recordRow.length; element++){
+         expect(recordRow[element].getText()).to.equal(expectedResult[element]);
+        }
+    });
+    it('11/LP-Should verify the record has checkBox', () => {
+        const icon = $("//tbody/tr/td/i");
+        expect(icon.getAttribute("class"))
+            .to.equal("fa fa-check-square-o");
+    });
+});
```
[the remark this answers is not in the document]
Comment (MariaMoroz) [reply test/specs/us001_01.js:90]: done

**Pull request closed without merging**